File: Backend/App/models/physical_generals.py
from sqlalchemy import Column,Integer,String,Boolean,Date,ForeignKey
from Backend.App.core.database import Base
from Backend.App.models.case_history import CaseHistory

class PhysicalGenerals(Base):
    __tablename__="Physical_generals"

    PhysicalGeneralsId = Column(Integer,primary_key=True,index=True)
    CaseId = Column(Integer,ForeignKey(CaseHistory.CaseId))
    Appetite = Column(String(100))
    Thirst = Column(String(100))
    Sleep = Column(String(255))
    Stool = Column(String(255))
    Urine = Column(Date)
    Sweat = Column(Integer)

File: DotNetBackend/ViveksHomoeoClinic/ViveksHomoeoClinic/Models/EFModel/PhysicalGenerals.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ViveksHomoeoClinic.Models.EFModel
{
    public class PhysicalGenerals
    {
        [Key]
        public int PhysicalGeneralsId { get; set; }
        [Required]
        public int CaseId { get; set; }

        [ForeignKey(nameof(CaseId))]
        public CaseHistory Case { get; set; }
        public string Appetite { get; set; }
        public string Thirst { get; set; }
        public string Sleep { get; set; }
        public string Stool { get; set; }
        public string Urine { get; set; }
        public string Sweat { get; set; }
    }
}
